libcomps/src/comps_log_codes.h

```diff
@@ -48,43 +48,16 @@
 
 void __expand(char *str, const char *fmt, char out, ...);
 
-inline void expand0(char *str, const char *fmt, char **args, char out) {
```

Comment:
Another possibility might be simply switching these `inline` functions to `static inline` in this header? But I guess we expose these `expandN` functions externally, right?

Reply:
Indeed they are exposed. They are part of the installed headers: https://github.com/rpm-software-management/libcomps/blob/master/libcomps/src/CMakeLists.txt#L22

@jan-kolarik thank you for looking into this!